data/schedule.yaml

```diff
@@ -7,9 +7,21 @@
   - title: Check In
     start: 08:00
   - title: Opening
-    speaker: Organizers
-    start: 8:30
-    end: 9:30
+    Hosts: Ana, Zhiwei
+    start: 9:00
+    end: 9:05

```

Comment:
Talks need minimum 2 minutes between to swap speakers, usually we space 5 minutes.